syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -987,7 +987,12 @@ def simplify(worker: AbstractWorker, tensor: "AdditiveSharingTensor") -> tuple:
         # Don't delete the remote values of the shares at simplification
         tensor.set_garbage_collect_data(False)
 
-        return (tensor.id, tensor.field, tensor.crypto_provider.id, chain)
+        return (
+            tensor.id,
```

Comment:
You add a simplifier for the object_pointer id, but not here
Any reason for this?